### PR TITLE
Updated process handling

### LINKED ISSUE #23: Add process status information
Right now, we only display the processes that are running (indeed, there are no checks if the process is running or not). We need to provide historical and accurate information for the processes unless the user intentionally removes them.

### LINKED ISSUE #32: Add unit tests for `HazelcastProcessStore`
Tests are missing for `HazelcastProcessStore` after the testing structure update.

### LINKED ISSUE #37: Better `member list` command
Current version only lists the member id's; we need additional values...

Should list all running hazelcast members on the local machine. Output will have mostly static information (except process status):

- name
- pid
- process status (e.g., RUNNING... see #23 )
- cluster name
- process start time / uptime

### PR DESCRIPTION
**Pull request by alparslanavci**
Some fixes & refactorings regarding process handling mechanism.

This fixes #23, fixes #32, and fixes #37.

### CODE REVIEW

**Comment by alparslanavci** [comment]
This fixes #23, fixes #32, and #37.

File: src/main/java/com/hazelcast/commandline/member/MemberCommandLine.java [hunk 1]
```diff
@@ -250,21 +257,18 @@ private void printProcessHeader(Map<String, HazelcastProcess> processes) {
...
         int pid = process.getPid();
         String processName = process.getName();
         if (namesOnly) {
             if (!runningOnly || process.getStatus() == RUNNING) {
                 println(processName);
             }
-        } else if (process.getStatus() != STOPPED) {
+        } else if (runningOnly && process.getStatus() == RUNNING) {
             printf(LIST_FORMAT,
                     processName, pid, process.getStatus(), process.getCreationInstant(), process.getClusterName());
```
Comment (mesutcelik) [inline src/main/java/com/hazelcast/commandline/member/MemberCommandLine.java]: The format for CREATED is not really human readable. Can we format this?
`2019-09-23T11:31:51.121Z`

Comment (alparslanavci) [reply src/main/java/com/hazelcast/commandline/member/MemberCommandLine.java]: 👍

**Comment by alparslanavci** [comment]
verify

**Pull request merged**